### ISSUE TITLE
Instructor: feedback results: provide copy instructions for Mac OS

### ISSUE THREAD

**Issue by chowyb** [TEAMMATES/teammates#7350]
In instructorFeedbackResultsPage, when using the "click here" text to view the results in a table, instructions are provided to copy and paste the table into an Excel workbook.
![feedbacktable](https://cloud.githubusercontent.com/assets/7546744/26298119/8c2b68f0-3f08-11e7-899b-970f0ed3a5b1.png)
However, Mac systems use `Command + C` and `Command + V` instead, which could be added to the instructions.

**Comment by arjunsuri** [comment]
hey I would like to take this issue

**Comment by arjunsuri** [comment]
Is this issue still open?

**Comment by arjunsuri** [comment]
Is it still open?

**Comment by ncbaihly** [comment]
It seems that the previous PR for this issue got closed and wasn't accepted. I'm going to start working on this as my first time PR.

**Comment by ncbaihly** [comment]
So I I made the changes(see screenshot), however they break some of the UI tests because the page doesn't match what the tests are expecting.

I'm going to go ahead and change the test files to match the changes. If everything works out I'll make a pull request soon.

**Comment by ncbaihly** [comment]
So I'm still working on fixing the UI tests, one issue I've run up against is that even though I checked the HTML files that are being compared with the pages from my local instance and amended them to match the instruction changes, I'm still failing tests.

The reason seems to be an extra newline, but when I check the corresponding HTML file that's being compared with what the site displays, there is no newline at the position that failed. I tried clearing the browser cache, but that did not do anything.

Is there somewhere else where things might be cached that could affect the tests? Or is the HTML file transformed at all before being compared? Looking at the verifyHtml functions it doesn't seem to be, but I wonder if I might be missing something.

**Comment by ncbaihly** [comment]
@LiHaoTan Oh I am not, I must have missed that when I was reading the starter's guide. That sounds like it well help with my issue, Thanks!

**Comment by LiHaoTan** [comment]
@ncbaihly I just went to revisit the documentation and I think we weren't very clear about when to utilize GodMode. If you have any suggestions on how to improve the documentation do let us know :)

**Comment by ncbaihly** [comment]
@LiHaoTan I went back and took a look, maybe this would be more relevant posted somewhere else, but I think it may be a good idea to add a section about GodMode to the Development Workflow(process.md) document, which is what I've used a lot in referencing to try and make sure I'm following the correct process. 

Maybe add a little bit about it and a link to its documentation page in Step 3, subpart 3. I think adding in a bit about how you might need to update stuff with GodMode in the "Before submitting your work for review, here are some things to check (non-exhaustive):" section might be helpful.
